<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Language Translate</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>Images/IMG-TranslateForm.png</normaloff>Images/IMG-TranslateForm.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(39, 55, 70);</string>
  </property>
  <widget class="QTextEdit" name="textFirstLanguage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>441</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:insert;
border:1px solid;
border-color:rgb(88, 214, 141);
color:white;
</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textLastLanguage">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>441</width>
     <height>351</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:insert;
border:1px solid;
border-color:rgb(88, 214, 141);
color:white;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnTranslate">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>410</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  		border-style:insert;	
		border:1px solid;
		background-color:rgb(40, 116, 166);
		color:white;	
}

QPushButton:hover
{
   		background-color:rgb(46, 134, 193);
}

QPushButton:Pressed
{
		background-color:rgb(52, 152, 219);
}</string>
   </property>
   <property name="text">
    <string>Traducir</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/IMG-Ttranslate.png</normaloff>Images/IMG-Ttranslate.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnPaste">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string>QPushButton
{
  		border-style:insert;	
		border:1px solid;
		background-color:rgb(40, 116, 166);
		color:white;	
}

QPushButton:hover
{
   		background-color:rgb(46, 134, 193);
}

QPushButton:Pressed
{
		background-color:rgb(52, 152, 219);
}</string>
   </property>
   <property name="text">
    <string>Pegar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/IMG-Paste.png</normaloff>Images/IMG-Paste.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnClean">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>410</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string>QPushButton
{
  		border-style:insert;	
		border:1px solid;
		background-color:rgb(40, 116, 166);
		color:white;	
}

QPushButton:hover
{
   		background-color:rgb(46, 134, 193);
}

QPushButton:Pressed
{
		background-color:rgb(52, 152, 219);
}</string>
   </property>
   <property name="text">
    <string>Limpiar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/IMG-Clear.png</normaloff>Images/IMG-Clear.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="CboFirstLanguage">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:insert;
border:1px solid;
border-color:rgb(52, 152, 219);
background-color:rgb(39, 55, 70);
color:gray;
</string>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnCopy">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>410</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
  		border-style:insert;	
		border:1px solid;
		background-color:rgb(40, 116, 166);
		color:white;	
}

QPushButton:hover
{
   		background-color:rgb(46, 134, 193);
}

QPushButton:Pressed
{
		background-color:rgb(52, 152, 219);
}</string>
   </property>
   <property name="text">
    <string>Copiar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/IMG-Copy.png</normaloff>Images/IMG-Copy.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="CboLastLanguage">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style:insert;
border:1px solid;
border-color:rgb(52, 152, 219);
background-color:rgb(39, 55, 70);
color:gray;
</string>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BtnChange">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Century Gothic</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
		border-style:insert;	
		border:1px solid;
		background-color:rgb(245, 176, 65);
}

QPushButton:hover
{
   background-color:rgb(248, 196, 113);
}

QPushButton:Pressed
{
		background-color:rgb(249, 231, 159);
}</string>
   </property>
   <property name="text">
    <string>Cambiar</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Images/IMG-Trasfer.png</normaloff>Images/IMG-Trasfer.png</iconset>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
